node/runtime/Cargo.toml

```diff
@@ -30,7 +30,7 @@ authorship = { package = "srml-authorship", path = "../../srml/authorship", defa
 babe = { package = "srml-babe", path = "../../srml/babe", default-features = false }
 balances = { package = "srml-balances", path = "../../srml/balances", default-features = false }
 collective = { package = "srml-collective", path = "../../srml/collective", default-features = false }
-contracts = { package = "srml-contracts", path = "../../srml/contracts", default-features = false }
+contracts = { package = "srml-contracts", path = "../../srml/contracts", default-features = false, features = ["core"] }
```

Comment:
Please remove this `core` feature from `srml-contracts`. There is no reason we have this feature anymore.